packages/ruleset/src/functions/index.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
   securitySchemeAttributes: require('./securityscheme-attributes'),
   securitySchemes: require('./securityschemes'),
   stringAttributes: require('./string-attributes'),
+  unevaluatedPropertiesCheck: require('./unevaluated-properties'),
```

Comment:
How would you feel about, for new rules, keeping the function associated with a rule the same as the rule name? My reasoning doesn't apply so much to this rule but for others, the discrepancy has made it difficult for me to match up "functions" with rules.

Reply:
so for this one, should we call it `ibmUnevaluatedProperties` or just `unevaluatedProperties`?   I would lean toward not using the "ibm" prefix to keep the function names short.

Reply:
Oh, good clarification - I meant `unevaluatedProperties`. I agree, the "ibm" part would be unnecessary

Reply:
done